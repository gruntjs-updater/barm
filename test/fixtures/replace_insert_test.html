<!--insert
        <script type="text/javascript" src="vendors/release/dojo/dojo-mini.js" ></script>
        <script type="text/javascript" src="vendors/release/dijit/dijit-mini.js" ></script>
        <script type="text/javascript" src="vendors/release/dojox/dojox-mini.js" ></script>
/insert-->

<!--replace to=^<script type="text/javascript" src="vendors/release/dojox/dojox-mini.js" ></script>^-->
<script type="text/javascript" src="vendors/release/dojox/dojox-mini.js" ></script>
<!--/replace-->

<!--replace to=^/release/^ pattern=^/dev/^-->
<link rel="stylesheet" type="text/css" href="vendors/dev/dijit/themes/tundra/tundra.css">
<link rel="stylesheet" type="text/css" href="vendors/dev/dojox/grid/resources/Grid.css">
<link rel="stylesheet" type="text/css" href="vendors/dev/dojox/image/resources/Lightbox.css">
<!--/replace-->