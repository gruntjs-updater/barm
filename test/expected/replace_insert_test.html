
        <script type="text/javascript" src="vendors/release/dojo/dojo-mini.js" ></script>
        <script type="text/javascript" src="vendors/release/dijit/dijit-mini.js" ></script>
        <script type="text/javascript" src="vendors/release/dojox/dojox-mini.js" ></script>



<script type="text/javascript" src="vendors/release/dojox/dojox-mini.js" ></script>



<link rel="stylesheet" type="text/css" href="vendors/release/dijit/themes/tundra/tundra.css">
<link rel="stylesheet" type="text/css" href="vendors/release/dojox/grid/resources/Grid.css">
<link rel="stylesheet" type="text/css" href="vendors/release/dojox/image/resources/Lightbox.css">
